requirements/tests.txt

```diff
@@ -1,4 +1,5 @@
 pytest==2.7.3
 pytest-pep8
 mock
-tox
\ No newline at end of file
+tox
+pathlib
```

Comment:
So, I have a quibble with this file. Is tox a test dependency and the rest are dependencies enumerated in tox, or are these test dependencies and tox is a different kind of requirement?